apps/include/shell/tash.h

```diff
@@ -28,6 +28,7 @@
 #define __APPS_INCLUDE_SHELL_TASH_H
 
 #include <tinyara/config.h>
+#include <apps/builtin.h>
```

Comment:
I think #ifdef CONFIG_BUILTIN_APPS is needed

Reply:
I think this is needed because of line 42.
The structure of built-in is also used for tash command.